raaghu-elements/src/rds-stepper/rds-stepper.tsx

```
@@ -1,405 +1,451 @@
+              style={{
+                background: "#336cff",
+                width: progressIndex,
+              }}
+            ></div>
+            {formTitles.map((formItem, idx, arr) => (
+              <>
+                {idx < arr.length - 1 ? (
+                  <span
+                    className="dot"
+                    style={{ left: `${parseFloat(dotPosition) * (idx + 1)}%` }}
+                  ></span>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)